src/object/obj_rpc.c

```diff
@@ -1072,12 +1063,6 @@ crt_proc_struct_daos_req_comm_in(crt_proc_t proc, crt_proc_op_t proc_op,
 		if (rc)
 			return rc;
 	}
```

Comment:
rc = crt_proc_memcpy(proc, proc_op, drci, sizeof(*drci));
this one looks enough, need not pack it one by one.